### PR TITLE
breadth first

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/tree/tree.js [hunk 1]
```diff
@@ -53,8 +53,43 @@ class BinaryTree {
...
+
+    return max;
+  }
+  ///////////////////////////////////////////////////////
+
+  breadthFirst(root) {
+    if (!root) throw new Error;
+    let array = [];
+    let currentNode = this.root;
+    array.push(currentNode.value);
```
Comment (adrianhuebner) [inline javascript/code-challenges/tree/tree.js:76]: It looks like there is an issue with the `.value` coming back as undefined.
When you have a chance to come back to this and get the test passing I will regrade for you!
